nixarr/bazarr/default.nix

```diff
@@ -87,6 +87,45 @@ in {
       after = ["network.target"];
       wantedBy = ["multi-user.target"];
 
+      preStart = mkIf nixarr.autosync (
+        let
+          configure-bazarr = pkgs.writeShellApplication {
+            name = "configure-bazarr";
+
+            runtimeInputs = with pkgs; [util-linux coreutils bash yq];
+
+            text = ''
+              cd ${cfg.stateDir}
+              mkdir -p config
+              API_KEY=$(cat ${nixarr.stateDir}/api-key)
```

Comment:
would be good to make this parameterizable to allow secret management by e.g. agenix